02week/pigLatin.js

```diff
@@ -10,11 +10,32 @@ const rl = readline.createInterface({
 
 function pigLatin(word) {
 
-  // Your code here
-
+  const translateWord = (word) => { // define translateWord with parameter word
+    const wordArray = word.toLowerCase().trim().split(''); // define an array: converts string to array and characters to lowercase and stores in array
+    const vowelsArray = ['a','e','i','o','u']; // define an array to hold all vowels for comparison
+    let wordArrayEnd = []; // define an array to store the last half of the word
```

Comment:
should be const

Reply:
why if it will change each time the code runs?